bionic/flow.py

```diff
@@ -487,12 +487,12 @@ def merge(self, flow, keep='error', allow_name_match=False):
         flow: Flow
             Any Bionic Flow.
 
-        keep: 'error', 'old', or 'new' (default: 'error')
+        keep: 'error', 'self', or 'arg' (default: 'error')
             How to handle conflicting entity names.  Options:
 
             * 'error': throw an ``AlreadyDefinedEntityError``
-            * 'old': use the definition from this builder
-            * 'new': use the definition from ``flow``
+            * 'self': use the definition from this builder
+            * 'arg': use the definition from ``flow``
```

Comment:
Should we mention the deprecated values of `old` and `new`?

Reply:
is it deprecated or just plain unsupported?

Reply:
I think we ought to continue supporting `old` and `new`, at least for the next few releases, just to stay backwards compatible.  Since the old tests are still passing, it looks like this code does that.

Reply:
still supports `old`, the docstring doesn't encourage it

Reply:
@jqmp for the tests, I'm thinking keep `test_keep_new(merge_tester)` and add `test_keep_arg(merge_tester)` to continue testing the supported `old` syntax

Reply:
Yeah, that makes sense to me.  You could also use `@pytest.mark.parametrize` to run the test for both values.